src/bentoml_cli/cloud.py

```diff
@@ -60,7 +58,9 @@ def login(endpoint: str, api_token: str, context: str) -> None:  # type: ignore
             raise CLIException("current organization is not found")
 
         ctx = CloudClientContext(
-            name=context,
+            name=shared_options.context
```

Comment:
Er, is this right?

Reply:
yes right, before the default value is `default_context_name`. now if the context is not passed, the default value is the same as b4